commands.js

```diff
@@ -23,6 +23,7 @@ const fs = require('fs');
 const MAX_REASON_LENGTH = 300;
 const MUTE_LENGTH = 7 * 60 * 1000;
 const HOURMUTE_LENGTH = 60 * 60 * 1000;
+const BLACKLIST_DURATION = 365 * 24 * 60 * 60 * 1000; // 1 year
 
```

Comment:
Isnt a year a bit Much? Maybe we should start a thread in the ro sub forum asking all the ro's what they think the best blacklist time would be.